Sources/MockingURLProtocol.swift

```diff
@@ -22,6 +22,7 @@ public final class MockingURLProtocol: URLProtocol {
             let response = HTTPURLResponse(url: mock.url, statusCode: mock.statusCode, httpVersion: Mocker.httpVersion.rawValue, headerFields: mock.headers),
             let data = mock.data(for: request)
         else {
+            // swiftlint:disable nslog_prohibted
```

Comment:
Why do we keep dead code?

Reply:
This is to ignore the SwiftLint warning generated. Read more about it here: https://github.com/realm/SwiftLint#disable-rules-in-code